src/gwt/src/org/rstudio/studio/client/workbench/views/source/SourcePane.java

```diff
@@ -242,6 +242,7 @@ public void onResize()
       manageChevronVisibility();
    }
 
+   @Override
```

Comment:
Super-ultra-nitpicky: There are a bunch of methods in this class that are missing `@Override`.

Reply:
Thanks, I added them in 👍